pkg/kritis/cryptolib/verifier.go

```diff
@@ -70,23 +81,32 @@ func NewVerifier(imageDigest string, publicKeySet []PublicKey) (Verifier, error)
 // VerifyAttestation verifies an Attestation. See Verifier for more details.
 func (v *verifier) VerifyAttestation(att *Attestation) error {
 	var (
-		err     error
-		payload []byte
+		err       error
```

Comment:
In go initialize local variables at time of first assignment with :=. the only common exception is if they are needed outside the scope where they are first assigned, in which case you should declare them as close to first assignment as possible.

Reply:
Done.